.taskcluster.yml

```diff
@@ -105,6 +105,19 @@ tasks:
               TRUST_LEVEL: ${trust_level}
             features:
               taskclusterProxy: true
```

Comment:
Nit: add `chainOfTrust: true` here too. Otherwise, the artifacts below aren't usable

Reply:
Good eye